### PR TITLE
The intial route shouldn't run its entrance animation

### PR DESCRIPTION
**Pull request by abarth**
Instead, it should enter the world fully formed.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/routes.dart [hunk 1]
```diff
@@ -385,6 +385,13 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
 
   // Internals
 
+  Performance createPerformance() {
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/routes.dart]: I think this should be in PageRoute, maybe even MaterialPageRoute.

**Comment by Hixie** [comment]
LGTM assuming travis is happy.

File: packages/flutter/lib/src/widgets/routes.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/widgets/routes.dart]: Yeah, I wasn't sure where to put it.

**Comment by Hixie** [comment]
you'll need to fix the page_forward_transitions_test.dart test too, in theory

**Pull request merged**